CHANGELOG.md

```diff
@@ -50,7 +50,28 @@
     const appID = receipt.appID;
     ```
 + `getProgram` is moved to `@algo-builder/runtime` from `@algo-builder/algob`.
-
++ `runtime.addAsset`, `runtime.addAssetDef` and `runtime.addApp` are deprecated. 
+Please use `runtime.deployASA`, `runtime.deployASADef` and `runtime.deployAdd` instead of the above functions.
```

Comment:
```suggestion
Please use `runtime.deployASA`, `runtime.deployASADef` and `runtime.deployApp` instead of the above functions.
```
```suggestion
Please use `runtime.deployASA`, `runtime.deployASADef` and `runtime.deployAdd` instead of the above functions.
```